openquake/sep/landslide/nowicki_jessee.py

```diff
@@ -0,0 +1,164 @@
+from typing import Union
+import numpy as np
+
+
+LANDCOVER_TABLE={
+    b"14": 0.91,
+    b"20": 0.88,
+    b"30": 0.78,
+    b"40": 0.68,
+    b"50": 0.30,
+    b"60": 1.77,
+    b"70": 1.71,
+    b"90": -1.26,
+    b"100": 1.50,
+    b"110": 0.68,
+    b"120": 1.13,
+    b"130": 0.79,
+    b"140": 1.03,
+    b"150": 0.54,
+    b"160": 2.34,
+    b"170": 1.19,
```

Comment:
The paper doesn't provide a coefficient for class 170. How do we get the value for this class?

Reply:
I was checking the description of 170 class and took the coefficient of the 'closest by description' class from table 3, but I will do one check and return back to this comment